pandas/tests/extension/sparse/test_sparse.py

```diff
@@ -0,0 +1,212 @@
+import pytest
+import pandas as pd
+import numpy as np
+
+from pandas.core.sparse.dtype import SparseDtype
+from pandas import SparseArray
+from pandas.tests.extension import base
+import pandas.util.testing as tm
+
+
+def make_data():
+    data = np.random.uniform(size=100)
+    data[2::3] = np.nan
+    return data
```

Comment:
It was raised elsewhere if this actually should contain missing data (but I suppose in case of SparseArray this is the default fill value, so maybe a bit different case)